src/Common/src/Interop/Unix/System.Native/Interop.HostEntry.cs

```diff
@@ -45,9 +45,6 @@ internal unsafe struct HostEntry
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetHostEntryForName")]
         internal static extern unsafe int GetHostEntryForName(string address, HostEntry* entry);
 
-        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetHostByName")]
-        internal static extern unsafe int GetHostByName(string address, HostEntry* entry);
-
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetHostByAddress")]
```

Comment:
Presumably this can be removed too?  Looks like you removed the implementation already.

Reply:
yep, @geoffkizer internally do you use "vs code" to develop on linux?

Reply:
I mostly use full VS on Windows, but I've used VS Code sometimes too.

Reply:
mmm...i don't understand well, is there a way to compile/test .net core "unix" on Windows?On guide docs i've only found "vs code" choice.